ansible_runner/runner_config.py

```diff
@@ -178,6 +180,18 @@ def prepare(self):
             self.env['ANSIBLE_STDOUT_CALLBACK'] = 'minimal'
         else:
             self.env['ANSIBLE_STDOUT_CALLBACK'] = 'awx_display'
+
+            if self.resource_profiling:
+                callback_whitelist = os.environ.get('ANSIBLE_CALLBACK_WHITELIST', '').strip()
+                self.env['ANSIBLE_CALLBACK_WHITELIST'] = callback_whitelist + (',' if len(callback_whitelist) else '') + 'cgroup_perf_recap'
+                self.env['CGROUP_CONTROL_GROUP'] = '{}/{}'.format(self.resource_profiling_base_cgroup, self.ident)
+                self.env['CGROUP_OUTPUT_DIR'] = self.private_data_dir + '/profiling_data'
+                self.env['CGROUP_FILE_NAME_FORMAT'] = '%(task_uuid)s-%(feature)s.%(ext)s'
+                self.env['CGROUP_OUTPUT_FORMAT'] = 'json'
+                self.env['CGROUP_CPU_POLL_INTERVAL'] = '0.25'  # TODO
```

Comment:
Do we intend to make this configurable on a first pass?

Reply:
I was thinking this might be nice to expose, too - will make the change

Reply:
done